radio/src/storage/yaml/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ elseif(PCB STREQUAL X7)
     set(YAML_GEN_OUTPUT storage/yaml/yaml_datastructs_tpro.cpp)
   elseif(PCBREV STREQUAL T20)
     set(YAML_GEN_OUTPUT storage/yaml/yaml_datastructs_t20.cpp)
+  elseif(PCBREV STREQUAL T20V2)
```

Comment:
Might be able to merge this with line above to be a bit more DRY? Just a thought if it works... not bothered either way. 
```cpp
  elseif((PCBREV STREQUAL T20) OR (PCBREV STREQUAL T20V2))
```